### PR TITLE
adds code changes for renaming download-file-for-random-read to read_…

### PR DESCRIPTION
**Pull request by ankitaluthra1**
code changes for renaming download-file-for-random-read to cache-file-for-range-read

### Link to the issue in case of a bug fix.
[link](https://b.corp.google.com/issues/315824292)

### Testing details
1. Manual (on local) - done
2. Unit tests - done
3. Integration tests - done

### CODE REVIEW

File: perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
       "config_file_flags_as_json": {
         "file-cache": {
           "max-size-in-mb": -1,
-          "download-file-for-random-read": true
+          "cache-file-for-range-read": true
```
Comment (sethiay) [inline perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json:21]: @Tulsishah please check if this change will work. If not please suggest the correct way.

File: internal/cache/file/cache_handler_test.go [hunk 2]
```diff
@@ -336,7 +336,7 @@ func (chrT *cacheHandlerTest) Test_GetCacheHandle_IfLocalFileGetsDeleted() {
 	ExpectTrue(strings.Contains(err.Error(), util.FileNotPresentInCacheErrMsg))
 }
 
-func (chrT *cacheHandlerTest) Test_GetCacheHandle_DownloadForRandom() {
+func (chrT *cacheHandlerTest) Test_GetCacheHandle_CacheWithRangeRead() {
```
Comment (sethiay) [inline internal/cache/file/cache_handler_test.go:339]: nit: CacheForRangeRead

Comment (ankitaluthra1) [reply internal/cache/file/cache_handler_test.go]: done

File: perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json (hunk 1 above)
Comment (Tulsishah) [reply perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json:21]: This will work.

**Review by sethiay — APPROVED** [review]

**Pull request merged**